addons/website/views/snippets/snippets.xml

```diff
@@ -481,9 +481,10 @@
     </div>
 
     <!-- Mobile display options -->
-    <div data-js="MobileVisibility" data-selector="section .row > div"
+    <div data-js="MobileVisibility" data-selector="section .row > div, section"
          data-exclude=".s_col_no_resize.row > div">
-        <we-checkbox string="Shown On Mobile" data-show-on-mobile="true" data-no-preview="true"/>
+        <we-button class="o_we_link o_we_mobile" data-show-on-mobile="true" data-no-preview="true"
+                   data-img="/website/static/src/img/snippets_options/mobile_invisible.svg|/website/static/src/img/snippets_options/mobile_visible.svg"/>
```

Comment:
@qsm-odoo 
Should I use a single attribute and split the content, or two attributes?

Reply:
Two seems better. "data-img" for the default one, "data-active-img" for the one when the button is active